pvlib/tests/test_pvsystem.py

```diff
@@ -1374,6 +1374,21 @@ def test_PVSystem_multi_scale_voltage_current_power(mocker):
         system.scale_voltage_current_power(None)
 
 
+def test_PVSystem_get_ac_sandia(cec_inverter_parameters, mocker):
+    inv_fun = mocker.spy(inverter, 'sandia')
+    system = pvsystem.PVSystem(
+        inverter=cec_inverter_parameters['Name'],
+        inverter_parameters=cec_inverter_parameters,
+    )
+    vdcs = pd.Series(np.linspace(0, 50, 3))
+    idcs = pd.Series(np.linspace(0, 11, 3))
+    pdcs = idcs * vdcs
+    pacs = system.get_ac('sandia', vdcs, pdcs)
```

Comment:
I don't understand why this test passed because `vdcs` and `pdcs` are swapped from their correct order. Fixed in #1150

Reply:
`get_ac` orders the arguments `p_dc, v_dc` because `v_dc=None` is needed to support `pvwatts`. The inverter functions generally order the arguments `v_dc`, `p_dc` which is unfortunate.